test/e2e/simple_bridge_and_bond.go

```diff
@@ -215,6 +219,7 @@ var _ = Describe("NodeNetworkState", func() {
 				for _, node := range nodes {
 					interfacesNameForNode(node).ShouldNot(ContainElement("bond1"))
 				}
+				resetDesiredStateForNodes()
```

Comment:
Can't we do it in module-wide (or even better, package-wide) AfterEach?

Reply:
well we don't control what afterEach come last..., prefer to do this way to be sure that this is the last thing is called.

Reply:
Fine with me. Would you just add a TODO to resetDesiredStateForNodes mentioning that we should drop it once Policy cleans up the state?